ci/lib/stage-test-examples.jenkinsfile

```diff
@@ -93,7 +97,7 @@ stage('test-examples') {
         } catch (Exception e) { }
     }
 
-    if ((os_release_id != "alpine") && (env.node_label != "graphene_22.04_5.19")) {
+    if (((os_release_id != "alpine") && (env.node_label != "graphene_22.04_5.19")) || (env.base_os != "ubuntu24.04")) {
```

Comment:
Add condition for alpine and ubuntu 24.04 into single check

Reply:
Modified